features/cli/build.feature

```diff
@@ -753,6 +753,14 @@ Feature: build 'apps' with CLI
       | Build ConfigMaps:\s+cmtest->. |
       | Build Secrets:\s+secrettest->.|
     And the "ruby-hello-world-1" build completed
+    #Bug 1669981 comment #1, the secreti will not included in the output image.
```

Comment:
typo should be `secret` instead of `secreti`

Reply:
lgtm otherwise